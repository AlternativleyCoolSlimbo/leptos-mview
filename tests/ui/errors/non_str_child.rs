use leptos_mview::view;

fn main() {
    view! {
        div { 3 }
    };
}
