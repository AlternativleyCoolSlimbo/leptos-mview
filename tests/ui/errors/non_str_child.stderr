error: only string literals are allowed in children
 --> tests/ui/errors/non_str_child.rs:5:15
  |
5 |         div { 3 }
  |               ^
